src/gui/ogr/qgsnewogrconnection.cpp

```diff
@@ -138,4 +139,9 @@ void QgsNewOgrConnection::on_btnConnect_clicked()
   testConnection();
 }
 
+void QgsNewOgrConnection::helpRequest()
+{
+  QgsHelp::openHelp( "managing_data_source/opening_data.html#connecting-to-ogr" );
```

Comment:
@DelazJ I don't think that we have documentation for this. You can reach this dialog when adding a vector layer, then select the database radio button when then New or Edit connection.

![ogr-connection](https://user-images.githubusercontent.com/163681/29139730-35f53424-7d48-11e7-99ef-6d39fcae5c25.png)